Marlin/src/lcd/extui/mks_ui/draw_touch_calibration.cpp

```diff
@@ -86,7 +86,7 @@ void lv_update_touch_calibration_screen() {
     // end calibration
     str = stage == CALIBRATION_SUCCESS ? GET_TEXT(MSG_CALIBRATION_COMPLETED) : GET_TEXT(MSG_CALIBRATION_FAILED);
     touch_calibration.calibration_end();
-    lv_big_button_create(scr, "F:/bmp_return.bin", common_menu.text_back, BTN_X_PIXEL * 3 + INTERVAL_V * 4, BTN_Y_PIXEL + INTERVAL_H + titleHeight, event_handler, ID_TC_RETURN);
+    lv_big_button_create(scr, "F:/bmp_return.bin", common_menu.text_back, 180, BTN_Y_PIXEL + INTERVAL_H + titleHeight, event_handler, ID_TC_RETURN);
```

Comment:
Is this button unique or following a similar layout to others? Can its position be expressed in terms of (updated names) `BTN_SIZE_X` and `INTERVAL_W`?